media/css/plugincheck/plugincheck.less

```diff
@@ -263,8 +263,100 @@ html[dir="rtl"] {
 #sec-plugin-uptodate {
     display: none;
 }
-.footer-newsletter-form {
-    padding: 40px 20px;
+#mozorg-newsletter-form,
+#newsletter-form {
+    width: 1300px / 2;
+    margin: 0 auto;
+    padding: 0 0 (@baseLine * 2) 0;
+    background: transparent;
+    border-bottom: none;
+    box-shadow: none;
+
+    h3 {
+        display: block;
+        float: none;
+        width: auto;
+        padding: (@baseLine * 2) 0;
+        color: @textColorSecondary;
+        .font-size(46px);
+        text-align: center;
+        margin: 0;
+        letter-spacing: -1px;
+    }
+
+    .form-contents {
+        width: 50%;
+        margin: 0;
+
+        input[type="email"] {
+            height: 32px;
+            padding: 5px 10px;
+            background: #fbf9f4;
+            border: 2px solid #e1ddd3;
+            border-radius: 7px;
+            -webkit-appearance: none;
+        }
+
+        ::-webkit-input-placeholder {
+           color: #e1ddd3;
+           .open-sans;
+           font-weight: bold;
+           opacity: 1;
+        }
+
+        ::-moz-placeholder {
+           color: #e1ddd3;
+           .open-sans;
+           font-weight: bold;
+           opacity: 1;
+        }
+
+        :-ms-input-placeholder {
+           color: #e1ddd3;
+           .open-sans;
+           font-weight: bold;
+           opacity: 1;
+        }
+    }
+
+    .form-submit {
+        width: 50%;
+        margin: 0;
+
+        input[type="submit"] {
```

Comment:
Should add `text-align: center;` for the button
![plugincheck-button](https://cloud.githubusercontent.com/assets/205591/10764221/488ea6da-7c8a-11e5-98d1-5e43ba62d80d.jpg)